### ISSUE TITLE
how to convert torch.mean when use torch.mean(x)

### ISSUE THREAD

**Issue by niuwenju** [NVIDIA-AI-IOT/torch2trt#478]
how to convert **torch.mean** when use `y = torch.mean(x)` not x.mean(dim=xxx)
